src/portal/package.json

```diff
@@ -16,7 +16,11 @@
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
     "build": "ng build --aot",
-    "release": "ng build --prod"
+    "release": "ng build --prod",
+    "generate_swagger_json": "node scripts\\convert-yaml-to-json.js",
```

Comment:
'\\'  maybe not work in some environments

Reply:
> “\” “\” maybe not work in some environments

Fixed